tests/test-cases/Group11-Upgrade/11-01-Upgrade.robot

```diff
@@ -41,7 +41,15 @@ Run Docker Checks
     Should Be Equal As Integers  ${rc}  0
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} ps -a
     Should Be Equal As Integers  ${rc}  0
-    Should Contain  ${output}  Exited (143)
+
+
+    ${status}=  Get State Of Github Issue  5653
+    Run Keyword If  '${status}' == 'closed'  Fail  Exit code check below needs to be updated now that Issue #5653 has been resolved
+    Run Keyword If  '${status}' == 'closed'  Should Contain  ${output}  Exited (143)
```

Comment:
If you want this to actually trigger you need to switch these two. Otherwise the test exits on first line.